### ISSUE TITLE
Make sure to have an annotated tag on each release

### ISSUE THREAD

**Issue by barmac** [zeebe-io/zeebe-modeler#226]
__What should we do?__

Make sure to have an annotated tag on each release. 

__Why should we do it?__

When I run `npm run release` for Zeebe Modeler 0.9.1 release, lerna created just an ordinary git tag instead of an annotated one. This caused Travis CI to skip the commit in the distro stage. I had to replace tag manually. @pinussilvestrus confirmed that he experienced the same issue.

**Comment by nikku** [comment]
What is an ordinary vs. an annotated tag? :thinking:

**Comment by barmac** [comment]
`git tag v0.9.1` (that's what probably `lerna` run with `npm run release`, [outcome](https://travis-ci.com/github/zeebe-io/zeebe-modeler/builds/162712434))  v.  `git tag -a v0.9.1` ([outcome](https://travis-ci.com/github/zeebe-io/zeebe-modeler/builds/162732654). In the second case you get a commit which contains only the commit message plus reference to tag.

I guess we could just fix it on CI. 🤷

**Comment by nikku** [comment]
I do not see the relevant difference.

In both cases, CI will pick up the new tag and build it. Right?

**Comment by pinussilvestrus** [comment]
Nope, that's the thing here. In the first scenario, CI does not trigger the distro builds. Maciej had to annotate the tag manually to properly work

**Comment by barmac** [comment]
OK let's clarify this issue. The problem was that I had to manually create a tag commit to have the release picked up by CI.

**Comment by barmac** [comment]
I updated the issue description.
